software/colanode.yml

```
@@ -0,0 +1,18 @@
+name: "Colanode"
+website_url: "https://colanode.com"
+source_code_url: "https://github.com/colanode/colanode"
+description: "Colanode is an open-source collaboration suite with real-time messaging, rich text pages, file management, and dynamic databases - built for offline work. An alternative to Slack and Notion."
```

One minor adjustment here
```suggestion
description: "Colanode is an open-source collaboration suite with real-time messaging, rich text pages, file management, and dynamic databases - built for offline work. (alternative to Slack, Notion)."
```